src/planning/src/planning/planning.cpp

```diff
@@ -36,29 +36,33 @@ Planning::Planning() : Node("planning") {
   smoothing_spline_coeffs_ratio_ = declare_parameter<float>("smoothing_spline_coeffs_ratio_", 3.0);
   smoothing_spline_precision_ = declare_parameter<int>("smoothing_spline_precision_", 10);
   mode = declare_parameter<std::string>("adapter", "fsds");
-
-  // State Estimation map Subscriber
-  this->track_sub_ = this->create_subscription<custom_interfaces::msg::ConeArray>(
-      "/state_estimation/map", 10, std::bind(&Planning::track_map_callback, this, _1));
-
-  // Vehicle Localization Subscriber
-  this->vl_sub_ = this->create_subscription<custom_interfaces::msg::VehicleState>(
-      "/state_estimation/vehicle_state", 10,
-      std::bind(&Planning::vehicle_localization_callback, this, _1));
+  using_simulated_se_ = declare_parameter<int>("use_simulated_se", 0);
 
   // Control Publishers
   this->local_pub_ =
-      this->create_publisher<custom_interfaces::msg::PathPointArray>("local_planning", 10);
+      this->create_publisher<custom_interfaces::msg::PathPointArray>("/path_planning/path", 10);
 
   // Publishes path from file in Skidpad & Acceleration events
   this->timer_ = this->create_wall_timer(
       std::chrono::milliseconds(100), std::bind(&Planning::publish_predicitive_track_points, this));
-
+  RCLCPP_INFO(this->get_logger(), "Using %s adapter", mode.c_str());
   // Adapter to communicate with the car
   this->adapter = adapter_map[mode](this);
+
+  if (!using_simulated_se_ || mode == "vehicle") {
+    // State Estimation map Subscriber
+    this->track_sub_ = this->create_subscription<custom_interfaces::msg::ConeArray>(
+        "/state_estimation/map", 10, std::bind(&Planning::track_map_callback, this, _1));
+
+    // Vehicle Localization Subscriber
+    this->vl_sub_ = this->create_subscription<custom_interfaces::msg::VehicleState>(
+        "/state_estimation/vehicle_state", 10,
+        std::bind(&Planning::vehicle_localization_callback, this, _1));
+  }
 }
 
 void Planning::track_map_callback(const custom_interfaces::msg::ConeArray &msg) {
+  RCLCPP_INFO(this->get_logger(), "Planning received %d cones", (int)msg.cone_array.size());
```

Comment:
Again, INFO might be too high of a logging level for callbacks of high frequency